backend/utils/authutils/tvault.go

```diff
@@ -0,0 +1,89 @@
+package authutils
+
+import (
+	"Tella-Desktop/backend/utils/constants"
+	"io"
+	"os"
+)
+
+// TVault helper methods -> move this to utils
+func WriteTVaultHeader(salt, encryptDBKey []byte) error {
```

Comment:
since we have the version in the header, and the version will likely change at some point in the future, it would be a reassurance to have a test that changes the version a few times and makes sure all of the remaining vault contents are unimpacted. it should not be a problem, but just in case